e2e/tests/upgrades/genesis_test.go

```diff
@@ -0,0 +1,159 @@
+package upgrades
+
+import (
+	"context"
+	"os"
+	"time"
+	"encoding/json"
+	"testing"
+
+	"github.com/cosmos/ibc-go/e2e/testconfig"
+	"github.com/cosmos/ibc-go/e2e/testsuite"
+	tmjson "github.com/tendermint/tendermint/libs/json"
```

Comment:
I think we can use [cometbft](https://github.com/cometbft/cometbft/tree/main/libs/json)?